Deploy-CAM.ps1

```diff
@@ -117,7 +117,7 @@ Param(
     $camManagementUserGroup = $null,
 
     [switch]$updateSPCredential,
-    [bool]$retrieveAgentState = $true,
+    [bool]$retrieveAgentState = $false,
     [bool]$showAgentState = $true
```

Comment:
should just make this false too even though it doesn't do anything

Reply:
I think true is better as default so custom only need to set retrieveAgentState is true and it shows the state (normally people like to see if the state is there)

Reply:
i think we want to be consistant between v1 and v2

Reply:
nevermind, looks liek thats what v2 does